packages/ibm-products/src/components/AddSelect/AddSelectSort.tsx

```diff
@@ -33,23 +37,13 @@ export const AddSelectSort = ({
         const opts = [
           {
             id: `${cur}-asc`,
-            itemText: (
-              <>
-                <ArrowUp size={16} />
-                {cur}
-              </>
-            ),
+            label: cur,
             direction: 'asc',
```

Comment:
I think this is what threw me off when I originally looked at this issue. It looks like the `direction` labels are hardcoded.

Since the original implementation used arrows, we probably didn’t flag this for g11n/i18n initially.

Reply:
We could alternatively open a followup issue for this improvement since in my previous discussions with @davidmenendez, we thought it would likely affect the API a little bit to provide a `directionLabel` prop or something.